internal/jobs/common.go

```diff
@@ -31,25 +38,26 @@ func finishWithSuccess(ctx context.Context, reservationId int64) {
 		ctx = copyContext(ctx)
 	}
 
+	// get details
 	rDao := dao.GetReservationDao(ctx)
 	reservation, err := rDao.GetById(ctx, reservationId)
 	if err != nil {
 		logger.Warn().Err(err).Msg("unable to update job status: get by id")
 		return
 	}
-	logger.Debug().Msgf("Job step: %d/%d", reservation.Step, reservation.Steps)
+	if reservation.Step == reservation.Steps {
+		logger.Info().Msgf("Finishing reservation with success at step %d/%d", reservation.Step, reservation.Steps)
+	} else {
+		logger.Error().Msgf("Finishing reservation with success at step %d/%d", reservation.Step, reservation.Steps)
+	}
```

Comment:
Feels weird but I want this to be logged into Glitchtip because a reservation was finished while expected amount of steps was not fulfilled. Could be a warning but we do not send warnings to Glitchtip.